package.json

```diff
@@ -47,7 +53,8 @@
   },
   "scripts": {
     "start": "./node_modules/.bin/grunt start",
-    "devSetup": "./node_modules/.bin/bower install && ./node_modules/.bin/grunt install"
+    "devSetup": "./node_modules/.bin/bower install && ./node_modules/.bin/grunt install",
```

Comment:
I recently got known that we don't need full path here.

> bower install && grunt install

will also work even if modules are not installed globally